Makefile

```
@@ -28,6 +28,11 @@ GITHUB_WEBHOOK_ENTRY_POINT = "GithubWebhook"
 GITHUB_PROCESSOR_NAME ?= "github_process-owl"
 GITHUB_PROCESSOR_ENTRY_POINT = "GithubProcess"
 
+GITHUB_CLIENT_NAME ?= "github_client-owl"
+GITHUB_CLIENT_ENTRY_POINT = "GithubClient"
+GITHUB_CLIENT_ID ?= "client-id"
```

Can we set it to an empty string and fail if it's not set? Currently it allows to deploy a function with an invalid client secrets which won't work anyway. So it's better to fail earlier. Can be done in a separate PR, if you prefer. It would be also useful for the webhook (GH secret).